app/components/list-databases/component.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  sortBy: ['handle:asc'],
+
+  arrayChanged: function() {
+    Ember.run.once(this, 'checkModelArray');
+  }.observes('model.[]'),
```

Comment:
It's my understanding that the `.observes` syntax has been replaced by `Ember.computed`, but should be the same. This is bound to changes on the length of the model, so it should be called every time anything is added or removed from `model`, which makes calling `checkModalArray` redundant, along with sending the action `redirect` first... why not send `forceRedirect` all in one go?

```javascript
arrayChanged: Ember.computed('model.[]', function() {
  if (this.get('model.length') === 0) {
    this.sendAction('forceRedirect');
  }
}),
```